source/MaterialXContrib/CMakeLists.txt

```diff
@@ -31,10 +26,18 @@ target_link_libraries(
     MaterialXRender
     ${CMAKE_DL_LIBS})
 
-install(TARGETS MaterialXContrib
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/
+target_include_directories(MaterialXContrib
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../>
+        $<INSTALL_INTERFACE:include>
```

Comment:
Using the MaterialXContrib target will now imply we need these include directories when used with a packaged MaterialX.